### ISSUE TITLE
training is steps is doubled...? 

### ISSUE THREAD

**Issue by Maki9009** [d8ahazard/sd_dreambooth_extension#1044]
Kindly read the entire form below and fill it out with the requested information.
Please find the following lines in the console and paste them below. If you do not provide this information, your
issue will be automatically closed.

Python 3.9.12 (main, Apr  4 2022, 05:22:27) [MSC v.1916 64 bit (AMD64)] Commit hash: 5be87ba63f62c228cf135425e21577f70c4e3351 Installing requirements for Web UI Skipping dreambooth installation. Dreambooth revision is The Latest Diffusers version is I'd like to assume its the latest also . Torch version is torch-1.13.1+cu117 Torch vision version is  torchvision-0.14.1+cu117..

Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?
yes

Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?
yes

Have you updated Dreambooth to the latest revision?
yes

Have you updated the Stable-Diffusion-WebUI to the latest version?
yes

No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.

Describe the bug

so when i train.. i have 13 images and i set num epochs to 150... so that should be 1950 steps... why is it then training to 3900? 

Provide logs

***** Running training *****
Num batches each epoch = 2
Num Epochs = 150
Batch Size Per Device = 13
Gradient Accumulation steps = 1
Total train batch size (w. parallel, distributed & accumulation) = 13
Text Encoder Epochs: 112
Total optimization steps = 1950
Total training steps = 3900
Resuming from checkpoint: False
First resume epoch: 0
First resume step: 0
Lora: False, Optimizer: 8Bit Adam, Prec: bf16
Gradient Checkpointing: False
EMA: True
UNET: True
Freeze CLIP Normalization Layers: False
LR: 2e-06
V2: False
Steps: 0%| | 0/3900 [00:00<?, ?it/s]

**Comment by ArrowM** [comment]
Step count is doubled if you use class pics. (it trains on the instance for 1 step, then the class for 1 step).
